ztp/source-crs/du-profile-kube-compare/optional/sriov-fec-operator/SriovFecClusterConfig.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: sriovfec.intel.com/v2
+kind: SriovFecClusterConfig
+metadata:
+  name: config
+  namespace: vran-acceleration-operators
+spec:
+  drainSkip: {{ .spec.drainSkip }} # true if SNO, false by default
+  priority: 1
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  acceleratorSelector:
+    pciAddress: $pciAddress
```

Comment:
Should this be templated as well? There are other occurrences for $mcp.

Reply:
Yeah thanks for the catch! Too bad I have dont have "full" cluster (which includes all the optional + required) to run all the rules in one shot. Will look around to see if someone can help with that